apps/web/vibes/soul/sections/error/index.tsx

```diff
@@ -1,20 +1,21 @@
-import { ButtonLink } from '@/vibes/soul/primitives/button-link';
-
-interface Link {
-  label: string;
-  href: string;
-}
+import { Button } from '@/vibes/soul/primitives/button';
 
 interface Props {
   title?: string;
   subtitle?: string;
-  cta?: Link;
+  ctaLabel?: string;
+  ctaAction?: () => void;
+}
+
+function defaultAction() {
+  window.location.reload();
 }
```

Comment:
I actually think this default action might be a problem since this file doesn't have `'use client'`. Any RSC that imports it would run into an issue if it doesn't add `use client`. An easy way around it would be to have a separate file for the form but we could also simply remove the default action and do nothing on click